x/dymns/types/string_list.go

```diff
@@ -0,0 +1,73 @@
+package types
+
+import "sort"
```

Comment:
feel free to put this kind of utility inside it's own sub package


but in this case it also seems that a lot of this functionality should be generic and maybe already exists in some existing package online? like generic set or sorted set?

Reply:
This can be done later, I added a TODO for that, as waiting for some party doing something would take time and blocking work. So why it would be done in local and move out later.